virt/virtual_machines/advanced_vm_management/virt-about-multi-queue.adoc

```diff
@@ -0,0 +1,25 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="virt-about-multi-queue"]
+= About multi-queue functionality
+include::_attributes/common-attributes.adoc[]
+:context: virt-about-multi-queue
+
+toc::[]
+
+Use multi-queue functionality to scale network throughput and performance on virtual machines (VMs) with multiple vCPUs.
+
+By default, the `queueCount` value, which is derived from the domain XML, is determined by the number of vCPUs allocated to a VM. Network performance does not scale as the number of vCPUs increases. Additionally, because virtio-net has only one Tx and Rx queue, guests cannot transmit or retrieve packs in parallel.
+
+[NOTE]
+====
+Enabling virtio-net multiqueue does not offer significant improvements when the number of vNICs in a guest instance is proportional to the number of vCPUs.
+====
+
+[id="virt-about-multi-queue-ki_{context}"]
+== Known limitations
+
+* MSI vectors are still consumed if virtio-net multiqueue is enabled in the host but not enabled in the guest operating system by the administrator.
+* Each virtio-net queue consumes 64 KiB of kernel memory for the vhost driver.
+* Starting a VM with more than 16 CPUs results in no connectivity if networkInterfaceMultiqueue is enabled (link:https://issues.redhat.com/browse/CNV-16107[CNV-16107]).
```

Comment:
Should `networkInterfaceMultiqueue` be in backticks? And maybe qualify by saying "the `networkInterfaceMultiqueue` field" or something?